src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -526,7 +534,11 @@
   []
   (let [^ContextHandlerCollection chc (ContextHandlerCollection.)]
     {:handlers chc
-     :state (atom {:endpoints {}})
+     :state (atom {:endpoints {}
+                   :mbean-container nil
+                   :overrides-read-by-webserver false
```

Comment:
I don't think `atom` even existed back then.  Besides, we were young, and needed the money.